tests/foreman/ui/test_computeresource_vmware.py

```diff
@@ -608,10 +607,9 @@ def test_positive_provision_end_to_end(
         }
     }
     with target_sat.ui_session() as session:
-        session.ansibleroles.import_all_roles()
-        assert session.ansibleroles.import_all_roles() == session.ansibleroles.imported_roles_count
         session.location.select(module_location.name)
         session.organization.select(module_sca_manifest_org.name)
+        assert session.ansibleroles.import_all_roles() == session.ansibleroles.imported_roles_count
```

Comment:
why not syncing just role you're using instead of syncing all?

Reply:
Yes, it is not necessary to sync all. Instead, I synced only a single role